scripts/generate-unable-to-act.ts

```diff
@@ -64,6 +64,18 @@ async function main() {
 		})
 	}
 
+    // The secondary effect from Moon Flute, Waning Nocturne, makes you
```

Comment:
VS Code's linter is complaining about this indentation being 4 spaces instead of a tab. Not sure why github passed it but we should probably fix it anyways.

Reply:
Ah, `yarn lint` only applies to `src/` apparently, so `scripts/` escaped notice.
Patched with a503c399d8a59fb52a530691fea6df9662acc4f7 and 5ee999fb

Probably worth patching package.json to include `scripts/`